### PR TITLE
Reduce RealNumberQ - take 4

### PR DESCRIPTION
**Pull request by rocky**
Go over `mathics.builtin.image.colors`n
    
Remove `ColorCombine` which does not work
Move pytests out of doctests

### CODE REVIEW

File: mathics/builtin/image/colors.py [hunk 1]
```diff
@@ -41,77 +41,112 @@ class Binarize(Builtin):
       <dd>map $t1$ < $x$ < $t2$ to 1, and all other values to 0.
     </dl>
 
-    S> hedy = Import["ExampleData/hedy.tif"];
-    S> Binarize[hedy]
+    >> hedy = Import["ExampleData/hedy.tif"];
```
Comment (rocky) [inline mathics/builtin/image/colors.py:44]: We no longer use temporary files. So this should be fine for sand boxing. 

@suhr - would you double check?

**Comment by mmatera** [comment]
> Go over `mathics.builtin.image.colors`n
> 
> Remove `ColorCombine` which does not work Move pytests out of doctests

Did it stop working recently, or maybe it never worked?

**Comment by mmatera** [comment]
Apart from the previous comment, LGTM

**Comment by rocky** [comment]
> > Go over `mathics.builtin.image.colors`n
> > Remove `ColorCombine` which does not work Move pytests out of doctests
> 
> Did it stop working recently, or maybe it never worked?

The current situation is that the arguments of [`ColorCombine`](https://reference.wolfram.com/language/ref/ColorCombine.html) are images.  In the commented out code, we have some sort of lists. 

You are in a better position than I to know what the history of this is. I note it is not in Mathematica 5.

**Pull request merged**